TESTS/Ace Deal Manager 2.1.1 - Default AFR - Workflow and updated test cases/Create new deal with add additional information like details, people, financial, products and activities - Critical/Steps/3/Input.html

```diff
@@ -1,73 +1,21 @@
 <div>
-  <strong>
-    <u>
-      <span style="font-size:small">
-        Adding addition details:-
-      </span>
-    </u>
-  </strong>
+  <strong><u>Adding additional details:-</u>
+  <em>Repeat 1.1 and 1.2 steps and do not save the deal, after that follow below steps :-</em></strong>
 </div>
-\n\n
 <div>
-  &nbsp;
-</div>
-\n\n
-<div>
-  <strong>
-    <em>
-      <span style="font-size:small">Repeat 1.1 and 1.2 steps and do not save the deal, after that follow below steps :-</span>
-    </em>
-  </strong>
-</div>
-\n\n
-<div>&nbsp;</div>
-\n\n
-<div>
-  <br />
-  \n
-  <span style="font-size:small">
-    1. Enter details ( Expected Close date and Project Alias ) in details sections.
-    <br />
-    \n
-    <br />
-    \n
-    2. Add team member, participants accounts and deal contacts in people section
-  </span>
-</div>
-\n\n
-<div>
-  \n
-  <ul>
-    \n 
-    <li><span style="font-size:small">Add 2-3 of each of the above</span></li>
-    \n
-  </ul>
-  \n
-  <br />
-  \n
-  <span style="font-size:small">
-    3. 
-  </span>
-  Add new products and enter the value in expected fees of product revenue section of each product and verify that that summation value of the field of all products&nbsp;is updated in \'Total Expected Fees\' in details sections.
-</div>
-\n\n
-<div>
-  <br />
-  \n
-  <span style="font-size:small">4. Add a product in product section</span>
-</div>
-\n\n
-<div>
-  \n
-  <ul>
-    \n 
+  <ol>
+    <li>Enter details ( Expected Close date and Project Alias ) in details sections.</li>
     <li>
-      <span style="font-size:small">Add 2-3 of each of the above</span>
+      Add team member, participants accounts and deal contacts in people section
+      <ul>
+        <li>Add 2-3 of each of the above</span></li>
+      </ul>
     </li>
-    \n
-  </ul>
-  \n
+  <li>Add new products and enter the value in expected fees of product revenue section of each product and verify that that summation value of the field of all products&nbsp;is updated in \'Total Expected Fees\' in details sections.</li>

```

Comment:
The instructions should be modified to include a save step - as it was unclear to me if the test was working or not because "Total Expected Fees" did not update until I saved.